pos_user_restriction/models/pos_config.py

```diff
@@ -0,0 +1,29 @@
+from odoo import api, fields, models
+
+
+class PosConfig(models.Model):
+    _inherit = "pos.config"
+
+    assigned_user_ids = fields.Many2many(
+        "res.users",
+        string="Assigned users",
+        help="Restrict some users to only access their assigned points of sale."
+        "In order to apply the restriction, the user needs the "
+        "'User: Assigned POS Only' group",
+    )
+    group_pos_user_id = fields.Many2one(
+        "res.groups",
+        compute="_compute_set_pos_user_group",
+        string="Point of Sale User Group",
+        help="This field is there to pass"
+        "the id of the pos user group to the point of sale client.",
+        store=True,
+    )
+
+    @api.depends("assigned_user_ids")
+    def _compute_set_pos_user_group(self):
```

Comment:
```suggestion
    def _compute_group_pos_user_id(self):
```

As per OCA guidelines, use `_compute_<field_name>` for compute method names

Reply:
Please check this comment as I think it hasn't been taken care of ☺️

Reply:
@ivantodorovich that suggestions are implemented  can you  Re-check it